stable/phpmyadmin/README.md

```diff
@@ -0,0 +1,80 @@
+```
+
+The command deploys Phpmyadmin on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the Phpmyadmin chart and their default values.
+
+|              Parameter               |               Description                |                         Default                         |
+|--------------------------------------|------------------------------------------|---------------------------------------------------------|
+| `image.registry`                     | PhpMyAdmin image registry                 | `docker.io`                                             |
+| `image.repository`                   | PhpMyAdmin Image name                     | `bitnami/phpmyadmin`                                     |
+| `image.tag`                          | PhpMyAdmin Image tag                      | `{VERSION}`                                             |
+| `image.pullPolicy`                   | Image pull policy                        | `Always` if `imageTag` is `latest`, else `IfNotPresent` |
+| `image.pullSecrets`                  | Specify image pull secrets               | `nil`                                                   |
+| `phpmyadmin.serviceType`            | type of service for PhpMyAdmin frontend             | `ClusterIP`                                                  |
+| `phpmyadmin.servicePort`        | port to expose service                   | `80`                                                   |
+| `phpmyadmin.dbPort`            | database port to use to connect                  | `3360`                                     |
+| `phpmyadmin.dbSuffix`                | Database suffix if included in the same release                  | `nil`                                          |
+| `phpmyadmin.dbHost`            | database host to connect to               | `nil`          |
+| `ingress.annotations`            | ingress annotations              | `{}`          |
+| `ingress.path`            | path to access frontend               | `/`          |
+| `ingress.host`            | ingress host               | `nil`          |
+| `ingress.tls`            | tls for ingress               | `[]`          |
+| `resources`                          | CPU/Memory resource requests/limits      | `{}`      |
+
+The above parameters map to the env variables defined in [bitnami/phpmyadmin](http://github.com/bitnami/bitnami-docker-Phpmyadmin). For more information please refer to the [bitnami/phpmyadmin](http://github.com/bitnami/bitnami-docker-Phpmyadmin) image documentation.
```

Comment:
Most of the parameters configure the kubernetes resource deployments. I think we should remove this line which says that the parameters map to env variables of the image.

Reply:
thanks